design/high-level-reactivity.md

```diff
@@ -223,6 +275,16 @@ A first-class developer friendly backend data driver will need to:
 
 While the ideal driver would emit exact deps and invalidations automatically for all queries, this is not likely to be practical for an arbitrary datastore. In practice, drivers will likely fall back to broader deps and invalidations, and tooling will help developers identify places where these invalidations can be optimized.  The developer can then rework their query or emit manual invalidations as needed.
 
+// How about the design of the Invalidation Server. Here's my ideas:
+ * It would be great if it's stateless, so if we need more load we can just scale it.
```

Comment:
How will being stateless make it easier to scale? Do you mean if we don't have the stateful websocket connection? Or do you mean if we don't persist the version numbers?